models.py

```diff
@@ -52,14 +52,16 @@ def forward(self, inputs: torch.Tensor, input_lengths: torch.Tensor):
 class Encoder(nn.Module):
 
 
-  def __init__(self, input_vocab_size, config: CfgNode, use_bilstm=False, glove_embeddings: Dict=None):
+  def __init__(self, input_vocab_size, lemmas_vocab_size, config: CfgNode, use_bilstm=False, glove_embeddings: Dict=None):
```

Comment:
I feel like if there are some new parameters to the Encoder which is used by Relation Identification too, there should be some changes there as well..? Is the RI module still working?
From what I remebered there should have been some changes in the train_heads_selection.py file, am I missing something?